### DISCUSSION TITLE
v1.7.6

### DISCUSSION THREAD

**Discussion by marcopeix** [Nixtla/neuralforecast#1212] [Announcements]
## New Features

- [FEAT]: Support providing DataLoader arguments to optimize GPU usage @jasminerienecker (#1186)
- [FEAT]: Set activation function in GRN of TFT @marcopeix (#1175)
- [FEAT]: Conformal Predictions in NeuralForecast @JQGoh (#1171)

## Bug Fixes

- [FIX]: Ability load models saved using versions before 1.7 @tylernisonoff (#1207)
- [FIX]: Conformal prediction issues @elephaint (#1179)
- [FIX]: Feature importance when using only hist_exog in TFT fails @elephaint (#1174)
- [FIX]: Remove unused output layer NBEATSx @elephaint (#1168)
- [FIX]: Fix Tweedie loss @elephaint (#1164)
- [FIX]: MLPMultivariate incorrect static_exog parsing @elephaint (#1170)
- [FIX]: Deprecate activation functions for GRU @marcopeix (#1198)

## Documentation

- [DOC]: Tutorial on cross-validation @marcopeix (#1176)
- [DOC]: Build docs on release only @elephaint (#1183)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/neuralforecast/releases/tag/v1.7.6'>v1.7.6</a>.</em>

**Comment by VivCh14** [comment]
After upgrading to this the `neuralforecast` lib does not work. I had a working model-training script that showed the tqdm progressbar when I trained the NHITS and BiTCN models but now after the upgrade, neither `numba` seems to be used nor the progress bar is shown and the model does not train. Has anyone had similar troubles ?
